components/SectionTitle.jsx

```diff
@@ -0,0 +1,127 @@
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import TooltipWrapper from './infoTooltip/TooltipWrapper';
+import LoadingMessage from './messages/LoadingMessage';
+import ErrorMessage from './messages/ErrorMessage';
+import ComingSoon from './messages/ComingSoon';
+import { createOnKeyDownHandler } from '../helpers/keyboardEventsHelper';
+
+require('../styles/components/_section-title.scss');
+
+const defaultControlledProps = {
+    isControlled: false,
+    toggleExpand: () => { },
+    isExpanded: false
+};
+
+const SectionTitle = ({
+    title,
+    icon,
+    children,
+    id = '',
+    classNames = '',
+    isCollapsible = false,
+    isComingSoon = false,
+    isLoading = false,
```

Comment:
I debated this, but I've decided I think `loading` / `error` etc don't really belong in this component. When we map components to sections from the website, the `children` will often be containers that fetch data and handle loading / error states. 
`isComingSoon` makes sense to stay. 
LMK what you think!